Let '-webkit-mask-box-image:initial' use the initial value

Loirooriol opened the pull request:
#### 20eb361ea0990e511cbdeb418f3a5c819a70afc4
<pre>
Let '-webkit-mask-box-image:initial' use the initial value
https://bugs.webkit.org/show_bug.cgi?id=249079

Reviewed by Tim Nguyen.

-webkit-mask-box-image:initial was setting it to is NinePieceImage(),
but the initial value is NinePieceImage(NinePieceImage::Type::Mask).

* LayoutTests/fast/css/getComputedStyle/getComputedStyle-webkit-mask-box-image-expected.txt: Added.
* LayoutTests/fast/css/getComputedStyle/getComputedStyle-webkit-mask-box-image.html: Added.
* Source/WebCore/css/CSSProperties.json:
* Source/WebCore/rendering/style/RenderStyle.h:
(WebCore::RenderStyle::initialNinePieceImageForMask):
(WebCore::RenderStyle::initialNinePieceImage): Deleted.

Canonical link: https://commits.webkit.org/257707@main
</pre>

[WebKit/WebKit@`5c0a579`](https://github.com/WebKit/WebKit/commit/5c0a579bd955ef857e5cf552962e0f850baff3b6)

nt1m commented on LayoutTests/fast/css/getComputedStyle/getComputedStyle-webkit-mask-box-image.html:
```diff
@@ -0,0 +1,81 @@
+<html>
+<head>
+<meta charset="utf-8">
+<script src="../../../resources/js-test-pre.js"></script>
```
nit: js-test-pre.js / js-test-post.js should not be used in new code. Please use `js-test.js` or better yet the WPT testharness.js file.

Loirooriol replied:
Changed it to use testharness.js

nt1m commented on LayoutTests/fast/css/getComputedStyle/getComputedStyle-webkit-mask-box-image.html:
```diff
@@ -0,0 +1,69 @@
+        for (let [prop, expected] of Object.entries(testCase[1])) {
+            assert_equals(computedStyle.getPropertyValue(prop), expected, prop);
+        }
+    }, JSON.stringify(testCase[0]));
```
Isn't testCase[0] already a string?

Loirooriol replied:
Yes, but in one case it's empty string, and then testharness would use the title of the file, which looks weird. So I thought I would just quote the strings.

nt1m replied:
sounds good

Loirooriol committed:
Let '-webkit-mask-box-image:initial' use the initial value
https://bugs.webkit.org/show_bug.cgi?id=249079

Reviewed by Tim Nguyen.

-webkit-mask-box-image:initial was setting it to is NinePieceImage(),
but the initial value is NinePieceImage(NinePieceImage::Type::Mask).

* LayoutTests/fast/css/getComputedStyle/getComputedStyle-webkit-mask-box-image-expected.txt: Added.
* LayoutTests/fast/css/getComputedStyle/getComputedStyle-webkit-mask-box-image.html: Added.
* Source/WebCore/css/CSSProperties.json:
* Source/WebCore/rendering/style/RenderStyle.h:
(WebCore::RenderStyle::initialNinePieceImageForMask):
(WebCore::RenderStyle::initialNinePieceImage): Deleted.

Canonical link: https://commits.webkit.org/257707@main
